tests/target/cargo-toml/basic/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+# comment before package
+[package]
+name = 3
+version = 1
+# comment 1  for arr1 line1
```

Comment:
Would it be possible to move these tests to `tests/target/configs/format_cargo_toml/true` and adding the `#rustfmt-format_cargo_toml: true` comment I mentioned above.